FileOrganization.md

```diff
@@ -1,5 +1,41 @@
 # File Organization
 
+## MARK Comments
```

Comment:
This section has some overlap with the Grouping section of this file. Could you merge them?
https://microsoft.github.io/swift-guide/FileOrganization.html#convention---grouping

Reply:
Good catch! Done.

Reply:
The existing Grouping section has a lot of overlapping information; could you add to that section instead?
For example, the image, and the explanation of the Xcode usages could be added to "Convention - Grouping" paragraph.

Reply:
I understand your point, but I would like to make sure the styling of these "MARK" comments is followed. It's something that needs to be very specific and handled correctly. Our team struggled with this for a long time. Making sure people know how to use them isn't just part of grouping. These are what even allow grouping to happen, and I'd argue grouping is a subset of them. 

I've corrected the structure though to match the rest. How does it seem now?

Reply:
I see. In that case "MARK comments" is indeed a better title. Thanks for following the pattern of "convention - rationale - examples"